krpc-code-gen/src/main/java/io/kroxylicious/krpccodegen/schema/MessageSpec.java

```diff
@@ -27,17 +27,21 @@ public final class MessageSpec {
 
     private final List<RequestListenerType> listeners;
 
+    private final Optional<Boolean> latestVersionUnstable;
+
     @JsonCreator
     public MessageSpec(@JsonProperty("name") String name,
                        @JsonProperty("validVersions") String validVersions,
                        @JsonProperty("fields") List<FieldSpec> fields,
                        @JsonProperty("apiKey") Short apiKey,
+                       @JsonProperty("latestVersionUnstable") Boolean latestVersionUnstable,
                        @JsonProperty("type") MessageSpecType type,
                        @JsonProperty("commonStructs") List<StructSpec> commonStructs,
                        @JsonProperty("flexibleVersions") String flexibleVersions,
                        @JsonProperty("listeners") List<RequestListenerType> listeners) {
         this.struct = new StructSpec(name, validVersions, fields);
         this.apiKey = apiKey == null ? Optional.empty() : Optional.of(apiKey);
+        this.latestVersionUnstable = latestVersionUnstable == null ? Optional.empty() : Optional.of(latestVersionUnstable);
```

Comment:
We use empty rather than `Optional.of(false)` here so we avoid emitting `latestVersionUnstable` in the[ template](https://github.com/kroxylicious/kroxylicious/pull/389/commits/6484ea9ae79d190fa5cdeca117b307b254d5939a#diff-56ae76c29f6440386434d83b99ec6235d4f30941b0c599f7a2917efe101693f0R30) right?

Reply:
This was mostly just following the existing pattern from `Optional<Short> apiKey()`. In the template you can then discriminate between unset and false.